### DISCUSSION TITLE
Pausing our own incentive program for the Tinyman's ASASTATS-ALGO LP providers

### DISCUSSION THREAD

**Discussion by ipaleka** [asastats/channel#191] [Announcements]
We just allocated funds for week 26 of our own [LP rewards incentive program](https://www.reddit.com/r/asastats/comments/slm2b4/asa_stats_is_officially_rolling_out_proposed_lp/). 

As you may already know, we've partnered with MetaLabs - you may farm your Tinyman's ASASTATS-ALGO LP tokens in their Cometa Liquidity Hub at https://app.cometa.farm/ from August 5th.

[As we announced](https://github.com/asastats/channel/discussions/190), our week 26 allocation was calculated with the end time before the launching of the Cometa program, you haven't lost your rewards if you staked your LP tokens immediately after the Cometa's launch.

As Cometa's program brings more rewards in ASASTATS than our incentive program and you'd be also rewarded with the same amount in ALGO, we've decided to pause our incentive program for the duration of Cometa's program (next 4 weeks). Please supply your LP tokens to the https://app.cometa.farm/farm instead.

It is yet to be discussed about the period after those 4 weeks and the eventual farming providers we may partner with. Whatever happens, bear in mind that our incentive programs would surely be powered again if there are no better options in a means of rewards quantity for our LP providers.

**Comment by ipaleka** [comment]
By u/MostlyPseudonymous on [Reddit](https://www.reddit.com/r/asastats/comments/widoxv/comment/ijatjlx/?utm_source=share&utm_medium=web2x&context=3):

Well then I'm out. I'm not going to be forced to interact with some new and untested Dapp just because all of a sudden you have a cozy relationship with them.

Good luck with the first exploit, everybody.

Tinyman is tested and quadruple audited and even it has been exploited. What a bone head move.

- - - - -

by hopemeetme: 

Everything in ASA Stats is to be discussed and everything in ASA Stats is directly influenced by the strength of our community members' opinions in a discussion.

You brought a valid and powerful opinion to this discussion so that means that if someone else doesn't counter you a lot we'll continue with our incentive program for the duration of Cometa's program.

Our relationship with Cometa is based on a pleasant meeting(s) AlgoRhythm and Babbexx had with them, as well as on the nice UI their platform has got - absolutely nothing else!

- - - - -

We have to discuss terms for this period when probably majority of LP providers would be on Cometa.

a) we can increase rewards for you who stay
b) we can allocate rewards for both you and those who locked their LP tokens on Cometa

This should be avoided IMO at any cost.

c) we can keep the same weekly amount to share for you who stay

Hm...

**Reply by ipaleka** [reply]
by u/MostlyPseudonymous:

Well, I do appreciate the response and that you are willing to be flexible as the community requests. But pleasant meetings and a nice UI are no basis to require that your LP providers risk their own money on a new DeFi platform which has yet to prove itself.

**Reply by ipaleka** [reply]
You're right, this announcement was by mistake shaped in a way that we kind of force you to use Cometa's platform rather than that we suggest you increase your rewards by using it.

Thank you for your comment(s)! We'll find a way to define an optimal amount of rewards for you who participate without locking your tokens, I guess there would be an announcement this week.

So, as things are now, Allocation week 27 is expected to happen the next Sunday!

**Reply by ipaleka** [reply]
> Our upper limit is 480K and that should stay, we shouldn't allocate more than that amount in total. We have a week in front of us to decide on actual amounts and schema.

Well, GamingToups system and @dragmz implementation are smarter than me: without any changes - and based on our incentive program's scheme - users will today share a 30% decreased total of 372K ASASTATS. :)

**Comment by ipaleka** [comment]
By Kenny_Bunkport on [Reddit](https://www.reddit.com/r/asastats/comments/widoxv/comment/ijbi48u/?utm_source=share&utm_medium=web2x&context=3):
·
Perhaps you should use a flash method and freeze the allocation amount to what was rewarded a week prior Comta being launched? IMO is the only way to be fair the participants who have abandoned the in house program for Cometa; as not to provide incentives for participants who choose not to use the Cometa. As we all know the individual rewards go up less and less as participants provide LPs in the in house rewards pool, using a snapshot of what it looks like now would be unfair.

- - - - -
By hopemeetme

That can bring additional problems, like not stimulating people to add more and not punishing them to reduce their participation... As well as not stimulating new providers I guess...

Our upper limit is 480K and that should stay, we shouldn't allocate more than that amount in total. We have a week in front of us to decide on actual amounts and schema.

**Reply by ipaleka** [reply]
By Kenny_Bunkport

Option C seems the only logical choice than . Keeping the same 480k allocated for thoes still Holding LPs, including the DEV pool (kinda solves the other [discussion](https://github.com/asastats/channel/discussions/60#discussioncomment-3340707) we were having). Keeping the in house program on, as is, throughout the 30 day period

- - - - -

By AlgoRhythMatic

I agree - option C… But based on the fact that a large part of the incentive to participate in this first 30-day trial is matching Algorand Foundation provided funds we don’t pay for. End users should of course not be forced to do anything, but the incentive to try an alternative option is available (and backed by the foundation).

- - - - -
By hopemeetme

Hm, keeping devpool in our incentive program basically acts only as leverage to decrease rewards for other participants as devpool won't keep any ASASTATS.

As opposite to Cometa's program, participating in it provides some ALGO that devpool can keep.

IMO we should act as planned - provide half of the tokens to Cometa.

**Reply by AlgoRhythMatic** [reply]
> including the DEV pool

Ah, I missed that portion of the statement!  Yes, I agree proceeding w/ half of DEV pool LP -> Cometa to take advantage of ALGO aspect during initial 30-day.

**Reply by ipaleka** [reply]
By Kenny_Bunkport

Taking a second look at it , I'm in agreement. I don't want to disadvantage anyone in the community. Everyone should be able to choose which program they want to participate in based on thier own individual risk tolerance.

**Reply by ipaleka** [reply]
> > including the DEV pool
> 
> Ah, I missed that portion of the statement! Yes, I agree proceeding w/ half of DEV pool LP -> Cometa to take advantage of ALGO aspect during initial 30-day.

Ok then, tomorrow we're going to supply half of the devpool tokens to Cometa.

**Comment by AlgoRhythMatic** [comment]
Outside of stopping/continuing our own incentive program, a key thing that should not be lost or forgotten here: Cometa's initial 30-day pools have the matching ALGO rewards provided by the Algorand Foundation, which cost us nothing.  We of course are not compelled to continue past this initial trial if we decide, as a community (ideally with **vocal participation** during planning phases), that some other path could yield better results - or offer better value for all.
